compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/FontRasterizationSettings.skiko.kt

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.text
+
+import androidx.compose.ui.text.platform.Platform
+import androidx.compose.ui.text.platform.currentPlatform
+
+/**
+ * Whether edge pixels draw opaque or with partial transparency.
+ */
+enum class FontEdging {
+    /**
+     * no transparent pixels on glyph edges
+     */
+    ALIAS,
+
+    /**
+     * may have transparent pixels on glyph edges
+     */
+    ANTI_ALIAS,
+
+    /**
+     * glyph positioned in pixel using transparency
+     */
+    SUBPIXEL_ANTI_ALIAS;
+}
+
+/**
+ * Level of glyph outline adjustment
+ */
+enum class FontHinting {
+    /**
+     * glyph outlines unchanged
+     */
+    NONE,
+
+    /**
+     * minimal modification to improve constrast
+     */
+    SLIGHT,
+
+    /**
+     * glyph outlines modified to improve constrast
+     */
+    NORMAL,
+
+    /**
+     * modifies glyph outlines for maximum constrast
+     */
+    FULL;
+}
+
+data class FontRasterizationSettings(
```

Comment:
[Don't use data classes in an API﻿](https://kotlinlang.org/docs/jvm-api-guidelines-backward-compatibility.html#don-t-use-data-classes-in-an-api)